Examples/ML/brain.py

```diff
@@ -0,0 +1,20 @@
+from sportsreference.nfl.teams import Teams
+from sklearn.linear_model import LinearRegression
+import numpy as np
+
+starting_year = 1970
+wins = []
+years = []
+ending_year = 2020
+
+for i in range(starting_year, ending_year):
+    teams = Teams(i)
+    lions = teams('DET')
+    wins.append([lions.wins])
+    years.append([i])
+
+years = np.array(years)
+wins = np.array(wins)
+
+LR = LinearRegression()
+LR.fit(years, wins)
```

Comment:
Similar to the first comment, I think it would be valuable to add outputs here and explain what the outputs mean/represent. Now that you have a trained model based on the team's wins per season, what would someone get with predicting output for new values? What insights can a user gain from this? These are just a few questions that someone might ask while reviewing this.